score.py

```diff
@@ -39,7 +39,9 @@ def glob_sample_data(sample, version):
         "post_qc_fasta":
             smart_glob(f"{sample}/results/{version}/gsnap_filter_[1,2].fa*", expected=2),
         "post_alignment_fasta":
-            smart_glob(f"{sample}/postprocess/{version}/taxid_annot.fasta", expected=1)
+            smart_glob(f"{sample}/postprocess/{version}/taxid_annot.fasta", expected=1),
+        "post_assembly_summary": # optional
+            smart_glob(f"{sample}/postprocess/{version}/assembly/*.hitsummary2.tab", expected=-1)
```

Comment:
I think it would be meaningful here to change expected from a number to a list of permissible values;  for optional files the list would be `expected=[0,2]`   I can do that later after you commit.